LDAR_Sim/model_code/generic_functions.py

```diff
@@ -291,3 +295,70 @@ def quick_cal_daylight(date, lat, lon):
     sunset = ss
 
     return (sunrise, sunset)
+
+def ecef_to_llh(ecef_km):
+    """
+    covert ecef cooridnate system to latitude, longitude and altitude
+    """
+    a = 6378.1370
```

Comment:
Either use descriptive variable name or a comment; What are these for?

Reply:
fixed